pkg/awsutils/awsutils.go

```diff
@@ -220,6 +221,26 @@ func prometheusRegister() {
 	}
 }
 
+//stringSet is a set of strings
+type stringSet struct {
```

Comment:
If we add the `reset()` function, we can replace `UnmanagedENISet` with this `stringSet`. Also, how will we ever remove a CIDR from the set otherwise?

Reply:
> If we add the `reset()` function, we can replace `UnmanagedENISet` with this `stringSet`.

I will open a separate issue for this one and do a follow up once we merge this PR.